diff --git a/node_modules/fractional/index.js b/node_modules/fractional/index.js
index 57a1bd8..b64817e 100644
--- a/node_modules/fractional/index.js
+++ b/node_modules/fractional/index.js
@@ -69,7 +69,7 @@ THE SOFTWARE.
  *      new Fraction('2 3/4') --> 11/4  (prints as 2 3/4)
  *
  */
-Fraction = function(numerator, denominator)
+var Fraction = function(numerator, denominator)
 {
     /* double argument invocation */
     if (typeof numerator !== 'undefined' && denominator) {
